### PR TITLE
Fix alias export for summarize_vars

### PR DESCRIPTION
**Pull request by Melkiades**

### CODE REVIEW

**Comment by pawelru** [comment]
Shall we use lifecycle deprecation message?

**Review by shajoezhu — APPROVED** [review]
lgtm! thanks @Melkiades

**Comment by Melkiades** [comment]
> Shall we use lifecycle deprecation message?

Done now. I added the current version, so we will have to wait. Note @BFalquet that using summarize_vars will generate a warning now

File: R/analyze_variables.R [hunk 1]
```diff
@@ -715,5 +716,36 @@ analyze_vars <- function(lyt,
     section_div = section_div
   )
 }
-
-summarize_vars <- analyze_vars
+#' @inherit analyze_vars
```
Comment (pawelru) [inline R/analyze_variables.R:719]: I think that a simple `@rdname analyze_vars` would be enough. Even without `@aliases`.

File: R/analyze_variables.R [hunk 2]
```diff
@@ -715,5 +716,36 @@ analyze_vars <- function(lyt,
...
+summarize_vars <- function(lyt,
```
Comment (pawelru) [inline R/analyze_variables.R:720]: out of curiosity:
can you do a simple:
```
summarize_vars <- function(...) {
  lifecycle stuff
  analyze_vars(...)
}
```

File: R/analyze_variables.R (hunk 1 above)
Comment (Melkiades) [reply R/analyze_variables.R:719]: Thanks, was looking for this

**Comment by pawelru** [comment]
Please add one more entry in the `_pkgdown.yaml` in the deprecated functions section

File: R/analyze_variables.R (hunk 1 above)
Comment (Melkiades) [reply R/analyze_variables.R:719]: Warning: Skipping 'analyze_vars.Rd'
ℹ File lacks name and/or title

Comment (Melkiades) [reply R/analyze_variables.R:719]: yep wrong name of the rd (analyze_variables). This probably need to change with the main function name

File: R/analyze_variables.R (hunk 2 above)
Comment (Melkiades) [reply R/analyze_variables.R:720]: done, much better
```r

## Select and format statistics output.
l <- basic_table() %>%
    split_cols_by(var = "ARM") %>%
    split_rows_by(var = "AVISIT") %>%
    summarize_vars(
        vars = "AVAL",
        .stats = c("n", "mean_sd", "quantiles"),
        .formats = c("mean_sd" = "xx.x, xx.x"),
        .labels = c(n = "n", mean_sd = "Mean, SD", quantiles = c("Q1 - Q3"))
    )

build_table(l, df = dta_test)
```

File: R/analyze_variables.R (hunk 1 above)
Comment (Melkiades) [reply R/analyze_variables.R:719]: I prefer describeIn so to add the deprecation badge

**Review by pawelru — APPROVED** [review]
good 👍  now you know how to deprecate functions :)

**Pull request merged**